### ISSUE TITLE
Display not working in some browsers

### ISSUE THREAD

**Issue by drkaputa** [miguelgrinberg/flask-video-streaming#38]
Hello,
   I was able to run your default code base a while back and it showed the video in a browser perfectly.  I couple days ago the videos stopped playing in all of my browsers on my computer. I thought I must have messed up the code somehow but the stream plays correctly on my phone browser.  Are you aware of any new updates to browsers that might be blocking your stream?  I also pulled down your latest code base and ran it to no avail on my PC browsers, but it indeed works on my phone browser.  Thank you for your time.
  ,Dan

**Comment by miguelgrinberg** [comment]
I'm not aware of any changes to Motion JPEG support in browsers.. Are you opening the web page or the direct stream URL? Many browsers do not support a direct streaming URL, but they are okay playing a stream embedded in a web page.

**Comment by drkaputa** [comment]
I pulled down your latest code from https://github.com/miguelgrinberg/flask-video-streaming and ran "python app.py".  I then entered http://192.168.0.17:5000 [IP address of my embedded computer running the code] into my other computer browser and saw "Video Streaming Demonstration" but no images.  I then entered the same IP address into my phone's browser and I see the top text and your 1,2,3 images as well.  On my PC I'm running win 7 with the latest firefox.  I believe I am opening the direct stream URL.  How would I embed the stream into a web page?

**Comment by miguelgrinberg** [comment]
No, you are opening the page that embeds the stream. The stream is an `<img>` tag in this page.

Unfortunately there is nothing I can do to help with this, it's up to the browsers to support this format.

**Comment by drkaputa** [comment]
?Thanks for the info Miguel.  Do you know any browsers that your code might work with? Or is there another way to visualize the video via VLC player?


  ,Dr. Kaputa

________________________________

**Comment by miguelgrinberg** [comment]
Chrome is what I tested this with at the time.
